### DISCUSSION TITLE
I am unable to upload my student ID card on Git Student.

### DISCUSSION THREAD

**Discussion by Ailson2023Jr** [community/community#113374] [GitHub Education]
### Hi everyone,

I am unable to upload my student ID card on Git Student.
![gitStudent](https://github.com/community/community/assets/126495403/96152666-caaa-42a6-be6e-81e526dd0ab0)

**Comment by santoshvandari** [comment]
To Upload the Student Card, You need to use the Mobile Browser. As far as I know, the system browser doesn't support it. You need to try from mobile devices. And hope it will work for you.

**Comment by samus-aran** [comment]
Hi @Ailson2023Jr 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
